### PR TITLE
PPC integration tests

### PR DESCRIPTION
**Pull request by schaefi**
**Added suse and fedora integration tests for Power**
   Simple disk image templates to run in KVM

**Update status helper script**
    Also show new ppc integration test builds

This is related to #1325

### CODE REVIEW

**Comment by schaefi** [comment]
* https://build.opensuse.org/project/show/Virtualization:Appliances:Images:Testing_ppc:suse

* https://build.opensuse.org/project/show/Virtualization:Appliances:Images:Testing_ppc:fedora

File: build-tests/ppc/fedora/test-image-vmx/config.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+#======================================
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# Mount system filesystems
+#--------------------------------------
+baseMount
```
Comment (davidcassany) [inline build-tests/ppc/fedora/test-image-vmx/config.sh]: I know this does not hurt, but I am confident this is not needed anymore (if needed it probably should be treated as a bug), the python code already ensure the mount points are ready.

File: build-tests/ppc/fedora/test-image-vmx/config.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+#======================================
+# Setup default target, multi-user
+#--------------------------------------
+baseSetRunlevel 3
+
+#======================================
+# Umount kernel filesystems
+#--------------------------------------
+baseCleanMount
```
Comment (davidcassany) [inline build-tests/ppc/fedora/test-image-vmx/config.sh]: The same here, python code already umounts whatever is needed the `config.sh` is executed.

File: build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# *REQUIRED*
```
Comment (davidcassany) [inline build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs]: Just curious to know from where you got this file or its contents. The rest of fedora config files here I have the feeling that with some patience and trials I could have produced them, but I am not that sure about this one. It is already provided by some package and just slightly patched?

File: build-tests/ppc/suse/test-image-vmx/config.sh [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+#======================================
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# Mount system filesystems
+#--------------------------------------
+baseMount
```
Comment (davidcassany) [inline build-tests/ppc/suse/test-image-vmx/config.sh]: Same as before

File: build-tests/ppc/suse/test-image-vmx/config.sh [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+#======================================
+# Remove yast if not in use
+#--------------------------------------
+suseRemoveYaST
+
+#======================================
+# Umount kernel filesystems
+#--------------------------------------
+baseCleanMount
```
Comment (davidcassany) [inline build-tests/ppc/suse/test-image-vmx/config.sh]: same as before

File: build-tests/ppc/fedora/test-image-vmx/config.sh (hunk 1 above)
Comment (schaefi) [reply build-tests/ppc/fedora/test-image-vmx/config.sh]: you are absolutely right. It was a left over from copying that stuff around. I deleted them also the exit 0 should not be there

File: build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs (hunk 3 above)
Comment (schaefi) [reply build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs]: I don't know either :) I copied those from a fedora description of the past

Comment (schaefi) [reply build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs]: maybe Neal has an idea if we still need it

@Conan-Kudo thoughts ?

Thanks

Comment (Conan-Kudo) [reply build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs]: Why do we need any of these? Shouldn't this be provided by Fedora packages already?

Comment (schaefi) [reply build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs]: To be honest I blindly copied the files as they were present on the other fedora images we have in the kiwi-descriptions repo. I could imagine that these files are provided by fedora packages but maybe the config is adapted for the needs of an image. I'm happy to reduce the number of overlay files, really

Comment (schaefi) [reply build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs]: So you are saying we better drop it ?

Comment (schaefi) [reply build-tests/ppc/fedora/test-image-vmx/root/etc/login.defs]: I dropped the file and will merge the test descriptions now. As they do not server a customer purpose we can adapt when needed.

**Pull request merged**